gef.py

```diff
@@ -8616,6 +8616,90 @@ def print_security_properties(self, filename):
         return
 
 
+@register_command
+class GotCommand(GenericCommand):
+    """Display current status of the got inside the process."""
+
+    _cmdline_ = "got"
+    _syntax_ = "{:s} [FUNCTION_NAME ...] ".format(_cmdline_)
+    _example_ = "got read printf exit"
+
+    def get_jmp_slots(self, readelf, filename):
+        output = []
+        cmd = [readelf, "--relocs", filename]
+        lines = gef_execute_external(cmd, as_list=True)
+        for line in lines:
+            if "JUMP" in line:
+                output.append(line)
+        return output
+
+    @only_if_gdb_running
+    def do_invoke(self, argv):
+
+        try:
+            readelf = which("readelf")
+        except IOError:
+            err("Missing `readelf`")
+            return
+
+        # get the filtering parameter.
+        func_names_filter = []
+        if argv:
+            func_names_filter = argv
+
+        # get the checksec output.
+        checksec_status = checksec(get_filepath())
```

Comment:
Can you move this block under the vmmap stuff, to keep it nearer the partial and norelro stuff?